docs/content-editing/creating-managing-and-reordering-blocks.md

```diff
@@ -0,0 +1,213 @@
+---
+myst:
+  html_meta:
+    "description": "Creating, managing, and reordering blocks"
+    "property=og:description": "Creating, managing, and reordering blocks"
+    "property=og:title": "Creating, managing, and reordering blocks"
+    "keywords": "Plone, Format, Block, Reorder"
+---
+# This guide offers a step-by-step approach to effectively create, manage, and reorder blocks within a Volto page.
+
+1.  In this example, you'll edit a page.
+
+    By default, the page includes a title block and the first text block.
+
+    To add another block, either press {kbd}`enter` or click on the {guilabel}`Add block` button <img alt="Add block icon" src="_static/circle-plus.svg" class="inline">.
+
+    ````{card}
+    ```{image} _static/creating-managing-and-reordering-blocks-01.jpeg
+    :alt: Click on the "+" to add another block
+    :target: _static/creating-managing-and-reordering-blocks-01.jpeg
+    ```
+    +++
+    _To add another block, click on the {guilabel}`Add block` button <img alt="Add block icon" src="_static/circle-plus.svg" class="inline">._
+    ````
+
+1.  Click on the type of block you want to add. For this example you'll add a few images and some text.
+        
+    ````{card}
+    ```{image} _static/creating-managing-and-reordering-blocks-02.jpeg
+    :alt: Add type of block you want
+    :target: _static/creating-managing-and-reordering-blocks-02.jpeg
+    ```
+    +++
+    _Add the type of block you want_
+    ````
+
+1.  Manage a block in the block itself. The Image block allows you to browse for an image or upload one.
```

Comment:
"image block" is not a proper noun, and its context is not a GUI label, so it should be lower case.

```suggestion
1.  Manage a block in the block itself.
    The image block allows you to browse for an image or upload one.
```